lib/ae_page_objects/element.rb

```diff
@@ -73,6 +73,17 @@ def using_default_locator?
       @locator == default_locator
     end
 
+    def reload_descendents
```

Comment:
The name `reload_descendents` and the comment seem to conflict with eachother.  "descendent" indicates down to me, but we're going up?  Maybe `reload_predecessors`, `reload_parents`, `reload_ancestors`?

Reply:
Good catch, I've renamed it to `reload_ancestors`